TestingApp/ViewController.swift

```diff
@@ -100,4 +120,11 @@ extension ViewController {
         })))
         present(alert, animated: true, completion: nil)
     }
+
+    func updateConf(with queryItems: [URLQueryItem]) {
```

Comment:
This can be easily called `updateConfiguration` too. Just some additional characters but a lot more readable

Reply:
fixed - 1a9f49f739cff96da5519abd665baa142882e0fe